.gitignore

```diff
@@ -83,3 +83,7 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+
+# macOS
+.DS_Store
+
```

Comment:
Looking at `braintree_android` gitignore we should probably include 
```
.gradle
/local.properties
/.idea
```
in the gitignore and remove the .idea files already included. There are a few other things in that .gitignore that I'm not sure if we need here or not yet.

Reply:
Right yeah this was pretty much *autogen*. I didn't modify much since I figure leaning in to Android studio might be for the best lol.